src/stores/ConfigStore.ts

```diff
@@ -232,6 +232,38 @@ export default class ConfigStore {
       }[];
       decodeText: string;
     }[] = [
+      {
+        asset: networkContracts.utils.dxdVestingFactory,

```

Comment:
what if utils.dxdVestingFactory is undefined? it can happen, we havet deploy those contracts in arbitrum yet. So you will have no value and it will break.